datastore/google/cloud/datastore/transaction.py

```diff
@@ -152,13 +153,19 @@ def Entity(*args, **kwargs):
 
     :type client: :class:`google.cloud.datastore.client.Client`
     :param client: the client used to connect to datastore.
+
+    :type options: `google.cloud.datastore.transaction.TransactionOptions`
+    :param options: the transaction options used for this transaction.
     """
 
     _status = None
 
-    def __init__(self, client):
+    def __init__(self, client, options=None):
         super(Transaction, self).__init__(client)
         self._id = None
+        if options is None:
+            options = TransactionOptions()
```

Comment:
`None` is an acceptable default for optional protobuf arguments (e.g. in GAPIC methods)